app/views/layouts/application.html.haml

```diff
@@ -28,7 +28,7 @@
     = theme_style_tags current_theme
     = vite_client_tag
     = vite_react_refresh_tag
-    = vite_polyfill_tag crossorigin: 'anonymous'
+    = vite_polyfills_tag crossorigin: 'anonymous'
```

Comment:
would it make sense to set `crossorigin` to default to anonymous? feels duplicative as it currently is